arch/arm/core/cortex_m/mpu/nxp_mpu.c

```diff
@@ -44,6 +48,19 @@ static inline u8_t _get_num_regions(void)
 	return FSL_FEATURE_SYSMPU_DESCRIPTOR_COUNT;
 }
 
+static inline u8_t _get_num_usable_regions(void)
+{
+	int max = _get_num_regions();
```

Comment:
_get_num_regions() returns an u8_t; please use it for the type of `max` to avoid confusions with signed/unsigned values, and implicit conversions between byte and int

Reply:
done